e2core/server/server.go

```diff
@@ -71,13 +71,8 @@ func New(l zerolog.Logger, sync *syncer.Syncer, opts *options.Options) (*Server,
 		logger:     ll,
 	}
 
-	if opts.AdminEnabled() {
-		e.POST("/name/:ident/:namespace/:name", server.executePluginByNameHandler(), auth.AuthorizationMiddleware(opts))
-	} else {
-		e.POST("/name/:ident/:namespace/:name", server.executePluginByNameHandler())
-		e.POST("/ref/:ref", server.executePluginByRefHandler(ll))
-		e.POST("/workflow/:ident/:namespace/:name", server.executeWorkflowHandler())
-	}
+	fmt.Printf("admin is enabled in e2core")
+	e.POST("/name/:ident/:namespace/:name", server.executePluginByNameHandler(), auth.AuthorizationMiddleware(opts))
```

Comment:
removed the "admin is not enabled" code path, and the branch that decided that

Reply:
Do we need/want that printf?

Reply:
Good catch! And nope, it was removed in https://github.com/suborbital/e2core/pull/422/commits/115ef7a55d1eaa51999106f85df5643f891861de